### DISCUSSION TITLE
What offers would you like to see in the GitHub Student Developer Pack?

### DISCUSSION THREAD

**Discussion by elisemoe** [community/community#17805] [GitHub Education]
We're always looking for new ways to support you in your classroom or journey as a student. What are the products, services, or tools you wish you could use but have been cost-prohibitive?

**Comment by pho3nix-writeups** [comment]
Hi @elisemoe, 

Thank you for the opportunity to share things I wish I could use but cannot afford. Hopefully, certifications are within the scope of tools. 
Here you go:

## Certifications:
- [Certified Ethical Hacker (CEH) Training & Certification](https://iclass.eccouncil.org/our-courses/certified-ethical-hacker-ceh/?utm_source=ecc-menu&utm_medium=eccreferral&utm_campaign=ceh-course-page#train)
- [Security+ Certification](https://www.comptia.org/certifications/security)

## Learn Hacking 
Some have student discounts but can add up.
- [Hack The Box Academy](https://academy.hackthebox.com/)
- [Hack The Box VIP](https://www.hackthebox.com/hacker/pricing#compare-plans-table)
- [TryHackMe Subscription](https://tryhackme.com/)

**Reply by Thorat2003** [reply]
What we have to do for swages??

**Comment by CyberFlameGO** [comment]
MATLAB is a big one I can think of

**Comment by flying1574** [comment]
AWS,GCP,AZ,Oracle for student

**Comment by Abir-Tx** [comment]
NI Multisim, PSpice these are needed too for students

**Comment by HiIamChaitanya** [comment]


**Reply by Lazylllama** [reply]
There is a Digital Ocean offer already :)

**Reply by acheong08** [reply]
Microsoft Azure doesn't require a credit card. However, it does ask for a lot more personal information and the only server location available for students is Australia.

**Comment by Sagar0-0** [comment]
Free access to Courses.

**Comment by Abdelkader-lahmar** [comment]
we need MATLAB
